app/services/api/school_partnerships/create_resource.rb

```diff
@@ -0,0 +1,122 @@
+module API::SchoolPartnerships
```

Comment:
Should we rename this to match the endpoint name? `Partnerships`?

Reply:
discussed in person with the team: we'll keep the controllers with API terms but everything else will have RECT terms, we'll revisit if necessary